test/models/service_area_test.rb

```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+describe ServiceArea do
```

Comment:
no sneaking in RSpec ;) we are using minitest for everything else, let's keep it one flavour

Reply:
good spot! i blame my muscle memory

Reply:
okay weird i just uncommented the demo test example in this file and it is valid minitest. checked i had removed rspec from my gemset and it still runs. i think it would be an idea to stick to one style of tests so everything is consistent. this policy should be documented (in the handbook) somehow?

Reply:
This has been removed